tests/frontend/unit/icui/chat/ChatMessage.test.tsx

```
@@ -76,4 +76,13 @@
-export {}; // Make this a module 
\ No newline at end of file
+import { describe, it } from 'vitest';
+
+// Placeholder suite to avoid Vitest "No test suite found" error
+describe.skip('ChatMessage Component', () => {
+	it('placeholder', () => {
+		// Implemented in future
+	});
+});
+
+export {}; // Make this a module
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)